src/service/download.service.ts

```diff
@@ -0,0 +1,33 @@
+/* eslint-disable class-methods-use-this */
+import { cwd } from 'process';
+
+import {
+  existsSync, mkdirSync,
+  writeFileSync, readFileSync,
+} from 'fs';
+
+const path = require('path');
+
+export class DownloadService {
+  private folder: string;
+
+  constructor() {
+    this.folder = path.resolve(cwd(), 'temp');
+  }
+
+  public async createFolder() {
```

Comment:
as you are using the `sync` versions of exist and mkdir, this functions does not have to be `async`